File: FizzBuzz.rb
#!/usr/bin/env ruby

puts "Please input a maximum integer for FizzBuzz test"
n = gets.chomp.to_i
puts "========================="
def fizz_buzz(n)
    for i in 0..n
      if i%3 == 0 and i%5 == 0
        puts "FizzBuzz"
      elsif i%3 == 0
        puts "Fizz"
      elsif i%5 == 0
        puts "Buzz"
      else
        puts i
      end
    end
end

fizz_buzz(n)

File: FizzBuzz.go
package main
import "fmt"
func main() {
	var number int
	fmt.Print("Enter maximum fizzbuzz value: ")
	fmt.Scan(&number)
	for i:=0; i<number+1;i++ {
		if i%5 == 0 && i%3 == 0 {
  			fmt.Println("FizzBuzz")
		}	else if i%3 == 0 {
				fmt.Println("Fizz")
		}	else if i%5 == 0{
				fmt.Println("Buzz")
		}	else{
				fmt.Println(i)
  }
}
}

	